public/mockServiceWorker.js

```diff
@@ -14,271 +14,271 @@ const IS_MOCKED_RESPONSE = Symbol('isMockedResponse')
 const activeClientIds = new Set()
 
 self.addEventListener('install', function () {
-  self.skipWaiting()
+	self.skipWaiting()
```

Comment:
no functional changes in this file, only formatting.

Reply:
nevermind, this gets generated afresh in CI by msw's postinstall hook, so we're better off ignoring it